### PR TITLE
Reply with 403 for manually blocked content

### PR DESCRIPTION
**Pull request by joaosa**
- So it's conformant with Badbits;
- Added a test to cover this;
- Further [ref](https://filecoinproject.slack.com/archives/C031K3BG527/p1678433424047959?thread_ts=1678232596.144739&cid=C031K3BG527)

p.s. disregard the PR branch name 😅

### CODE REVIEW

File: container/nginx/conf.d/shared.conf [hunk 1]
```diff
@@ -62,7 +62,7 @@ location / {
     # $arg_filename, $arg_download, $param_go_get, $http_if_none_match
     proxy_cache_key             $uri$format_final$arg_depth$arg_bytes$arg_filename$arg_download$param_go_get$http_if_none_match;
     proxy_cache_lock            on;
-    proxy_cache_valid           200 301 302 410     365d;
```
Comment (DiegoRBaquero) [inline container/nginx/conf.d/shared.conf:65]: Shouldn't we cache 403s then?

Comment (joaosa) [reply container/nginx/conf.d/shared.conf:65]: I supposed we could, but won't that interfere with other 403s on the shim codebase, such as [here](https://github.com/filecoin-saturn/L1-node/blob/main/container/shim/src/modules/registration.js#L234)? - I think this is the only place.

Comment (DiegoRBaquero) [reply container/nginx/conf.d/shared.conf:65]: Has it's own proxying config https://github.com/filecoin-saturn/L1-node/blob/51582eeba5b5453f1900978339ccf5289590c524/container/nginx/conf.d/register.conf

Comment (joaosa) [reply container/nginx/conf.d/shared.conf:65]: addressed

**Review by DiegoRBaquero — APPROVED** [review]

**Pull request merged**